prow/cmd/branchprotector/protect.go

```diff
@@ -97,6 +97,13 @@ func main() {
 	if err != nil {
 		logrus.WithError(err).Fatalf("Failed to load --config-path=%s", o.config)
 	}
+	branchWarns, repoWarns := cfg.BranchProtectionWarnings()
```

Comment:
Could we have a function that processes the warnings and outputs the warnings, so that it's shared between the two call-sites?

Reply:
Thank you for the review. Updated the PR.